src/libraries/System.Security.Cryptography.Xml/ref/System.Security.Cryptography.Xml.cs

```diff
@@ -60,8 +62,9 @@ public sealed partial class EncryptedData : System.Security.Cryptography.Xml.Enc
     public sealed partial class EncryptedKey : System.Security.Cryptography.Xml.EncryptedType
     {
         public EncryptedKey() { }
-        public string CarriedKeyName { get { throw null; } set { } }
-        public string Recipient { get { throw null; } set { } }
+        public string? CarriedKeyName { get { throw null; } set { } }
+        [System.Diagnostics.CodeAnalysis.AllowNullAttribute]
+        public string? Recipient { get { throw null; } set { } }
```

Comment:
This doesn't seem to be fixed: https://github.com/dotnet/runtime/pull/67198/files#r963817874